src/content/en/2024/security.md

```diff
@@ -2,17 +2,1740 @@
+      </tr>
+      <tr>
+        <td><code>cdn.userway.org</code></td>
+        <td class="numeric">6.20%</td>
+        <td class="numeric">5.69%</td>
+      </tr>
+      <tr>
+        <td><code>static.cloudflareinsights.com</code></td>
+        <td class="numeric">6.07%</td>
+        <td class="numeric">5.97%</td>
+      </tr>
+      <tr>
+        <td><code>code.jquery.com</code></td>
+        <td class="numeric">5.36%</td>
+        <td class="numeric">5.58%</td>
+      </tr>
+      <tr>
+        <td><code>cdn.jsdelivr.net</code></td>
+        <td class="numeric">3.92%</td>
+        <td class="numeric">4.27%</td>
+      </tr>
+      <tr>
+        <td><code>d3e54v103j8qbb.cloudfront.net</code></td>
+        <td class="numeric">2.15%</td>
+        <td class="numeric">1.88%</td>
+      </tr>
+      <tr>
+        <td><code>t1.daumcdn.net</code></td>
+        <td class="numeric">1.82%</td>
+        <td class="numeric">1.14%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Most common hosts from which SRI-protected scripts are included", sheets_gid="1612151810", sql_file="TODO.sql") }}</figcaption>
+</figure>
+
+Most of the hosts from which resources are fetched and protected by SRI are CDNs. A notable difference from 2022's data is the absence of `cdn.shopify.com` from the top hosts list (previously 22% on desktop and 23% on mobile). This is due to Shopify having dropped SRI in favor of similar functionality provided by the `integrity` attribute of `importmap`, which they explain in a <a hreflang="en" href="https://shopify.engineering/shipping-support-for-module-script-integrity-in-chrome-safari#">blogpost</a>.
+
+### Permissions Policy
+
+The [Permissions Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Permissions_Policy) (formerly known as the Feature Policy) is a set of mechanisms that allow websites to control which browser features can be accessed on a webpage, such as geolocation, webcam, microphone, and more. By using the Permissions Policy, websites can restrict feature access for both the main site and any embedded content, enhancing security and protecting user privacy. This is configured through the `Permissions-Policy` response header for the main site and all its embedded `<iframe>` elements,. Additionally, web administrators can set individual policies for specific `<iframe>` elements using their `allow` attribute.
+
+{{ figure_markup(
+  content="+1.3%",
+  caption="Relative increase in adoption of `Permissions-Policy` from 2022..",
+  classes="big-number",
+  sheets_gid="TODO",
+  sql_file="TODO.sql",
+) }}
+
+In 2022, the adoption of the `Permissions Policy` header saw a significant relative increase of 85%. However, from 2022 to this year, the growth rate has drastically slowed to just 1.3%. This is expected, as the Permissions Policy header was only introduced in 2021 and still solely supported by Chromium-based browsers under its current name, leading to an initial spike in adoption.
```

Comment:
IMHO this should either be "the Permissions Policy header" (without a dash, or code formatting as it's the name of the feature) or "the `Permissions-Policy` header" (with a dash and code formatting as it's the name of the actual header)

```suggestion
In 2022, the adoption of the `Permissions-Policy` header saw a significant relative increase of 85%. However, from 2022 to this year, the growth rate has drastically slowed to just 1.3%. This is expected, as the Permissions Policy header was only introduced in 2021 and still solely supported by Chromium-based browsers under its current name, leading to an initial spike in adoption.
```